src/Tasks/Microsoft.NET.Build.Tasks/PrepareForReadyToRunCompilation.cs

```diff
@@ -72,9 +71,6 @@ protected override void ExecuteCore()
                     Log.LogError(Strings.Crossgen5CannotEmitSymbolsInCompositeMode);
                     return;
                 }
-                
-                string isVersion6Preview1 = Crossgen2Tool.GetMetadata(MetadataKeys.IsVersion6Preview1);
```

Comment:
Please remove `MetadataKeys.IsVersion6Preview1` as well.

Reply:
Thanks for reminding me, removed in 2nd commit.

Reply:
Hmm, sorry, dropped the BOM again, will fix in a minute.

Reply:
Should be fixed now.